x/tradeshield/keeper/params.go

```diff
@@ -0,0 +1,16 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/elys-network/elys/x/tradeshield/types"
+)
+
+// GetParams get all parameters as types.Params
+func (k Keeper) GetParams(ctx sdk.Context) types.Params {
+	return types.NewParams()
```

Comment:
Should return from module KV Store with it's own Prefix Key

Reply:
same